packages/expo-image/android/src/main/java/expo/modules/image/ExpoImageView.kt

```diff
@@ -49,10 +49,27 @@ class ExpoImageView(context: ReactContext, private val requestManager: RequestMa
         }
     }
   }
+  private var propsChanged = false
```

Comment:
In general, I'm not a fan of mutable booleans for state/behavior stuff, because it's error-prone, but for now I don't mind keeping it.